inventory_management_system_api/repositories/manufacturer.py

```diff
@@ -1,6 +1,4 @@
-"""
-Module for providing a repository for managing manufacturers in a MongoDB database.
-"""
+"""Module for providing a repository for managing manufacturers in a MongoDB database."""
```

Comment:
Happy to do this if you think its best to be consistent, my personal preference would be to leave the one at the top of the file the same, only because it separates it out a bit from the text at the top of the editor
![image](https://github.com/ral-facilities/inventory-management-system-api/assets/90245114/55a138ef-cce9-4c0a-8c92-d9fc849d3709)
vs
![image](https://github.com/ral-facilities/inventory-management-system-api/assets/90245114/28f49b0f-2b65-4fcb-8e58-de0e65769bc5)

Reply:
I will revert these back then.